docs/spark/spark-procedures.md

```diff
@@ -275,6 +275,11 @@ See the [`RewriteDataFiles` Javadoc](../../../javadoc/{{% icebergVersion %}}/org
 and <br/> [`SortStrategy` Javadoc](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/actions/SortStrategy.html#field.summary)
 for list of all the supported options for this action.
 
+!!! Note
+    This procedure can also read the delete files created by merge-on-read mode if present
```

Comment:
How about remove 'created by merge-on-read mode'?  I don't think we ever say 'merge-on-read' mode anywhere in docs.